src/Bicep.Core/Registry/AzureContainerRegistryManager.cs

```diff
@@ -213,10 +223,13 @@ private static void ValidateBlobResponse(Response<DownloadRegistryBlobResult> bl
 
         private static async Task<Stream> ProcessLayer(ContainerRegistryContentClient client, OciDescriptor layer)
         {
-            if (!string.Equals(layer.MediaType, BicepMediaTypes.BicepModuleLayerV1Json, MediaTypeComparison))
+            string[] allowedMediaTypes = { BicepMediaTypes.BicepModuleLayerV1Json, BicepMediaTypes.BicepProviderArtifactLayerV1TarGzip };
+            // media types are case insensitive
+            if (!allowedMediaTypes.Contains(layer.MediaType, StringComparer.OrdinalIgnoreCase))
```

Comment:
>StringComparer.OrdinalIgnoreCase [](http://example.com/codeflow?start=61&length=32)

We have a pattern in this repo where we create a custom comparer or comparison object to make the code doing comparisons of that type of data more self documenting. You should keep the reference to a comparison or create a similar comparer. 

This also makes finding all the places easier via Find References in the code. (Another example is IdentifierComparer).